src/packagedcode/maven.py

```diff
@@ -1171,6 +1172,63 @@ def _parse(
         **urls,
     )
 
+class MavenPackageData(models.PackageData):
+
+    datasource_id = 'maven_pom'
+
+    def get_license_detections_for_extracted_license_statement(
+        extracted_license,
+        try_as_expression=True,
+        approximate=True,
+        expression_symbols=None,
+    ):
+        from packagedcode.licensing import get_normalized_license_detections
+        from packagedcode.licensing import get_license_detections_for_extracted_license_statement
+
+        if not MavenPackageData.check_extracted_license_statement_structure(extracted_license):
+            return get_normalized_license_detections(
+                extracted_license=extracted_license,
+                try_as_expression=try_as_expression,
+                approximate=approximate,
+                expression_symbols=expression_symbols,
+            )
+        
+        new_extracted_license = extracted_license.copy()
+        
+        for license_entry in new_extracted_license:
+            license_entry.pop("distribution")
+            if not license_entry.get("name", None):
```

Comment:
Nit: None is already the default for a get

Reply:
Also what about this instead:
`{key: value for key, value in license_entry.items() if value}`

Reply:
> Also what about this instead:
> {key: value for key, value in license_entry.items() if value and key != 'distribution'}
Actually we are doing a `pop()` so we actually not have empty key/value pairs in the original extracted license statement. Which is correct since we are creating these key/value pairs irrespective of them being present in the first place.